tests/test_script.py

```diff
@@ -53,3 +53,7 @@ def test_min_confidence():
     assert call_vulture([
         'vulture/core.py', '--exclude', 'whitelists',
         '--min-confidence', '100']) == 0
+
+
+def test_exclude():
+    assert call_vulture(['vulture/', '--exclude', 'vulture/core.py']) == 1
```

Comment:
What I meant was to exclude multiple individual files, i.e., in a comma-separated list.